modules/calib3d/test/test_usac.cpp

```diff
@@ -0,0 +1,326 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+
+namespace opencv_test {
+enum TestSolver { Homogr, Fundam, Essen, PnP};
+/*
+* @rng -- reference to random generator
+* @pts1 -- 2xN image points
+* @pts2 -- for PnP is 3xN object points, otherwise 2xN image points.
+* @two_calib -- True if two cameras have different calibration.
+* @K1 -- intrinsic matrix of the first camera. For PnP only one camera.
+* @K2 -- only if @two_calib is True.
+* @pts_size -- required size of points.
+* @inlier_ratio -- required inlier ratio
+* @noise_std -- standard deviation of Gaussian noise of image points.
+* @gt_inliers -- has size of number of inliers. Contains indices of inliers.
```

Comment:
Please remove `@` symbols (they are not necessary here)

Reply:
okay